openslides/users/models.py

```diff
@@ -158,11 +159,9 @@ def get_absolute_url(self, link='detail'):
         Returns the URL to the user.
         """
         if link == 'detail':
-            url = reverse('user_detail', args=[str(self.pk)])
+            url = "/users/%s/" % self.pk
         elif link == 'update':
-            url = reverse('user_update', args=[str(self.pk)])
-        elif link == 'delete':
-            url = reverse('user_delete', args=[str(self.pk)])
+            url = "/users/%s/edit/" % self.pk
```

Comment:
No. The update view has a template (a form). You can call this url in angular. But you can not call the delete url.